apps/src/javalab/constants.js

```diff
@@ -182,3 +182,8 @@ export const ExecutionType = {
   // Compile and run tests
   TEST: 'TEST'
 };
+
+export const ContentManagerType = {
+  // Javabuilder assets fetched via dashboard
+  DASHBOARD: 'dashboard'
```

Comment:
what will other values for `ContentManagerType` be?

Reply:
Yeah I took a closer look at the prototype, thinking boolean flag might make more sense. Just `useContentManager` (false now)?

Reply:
I was thinking that our current setup would be refactored into the contentmanager interface as well, but that does not look like the plan

Reply:
yeah a flag seems reasonable for now